cmd/up/dependencies.go

```diff
@@ -43,15 +43,14 @@ func downloadSyncthing() error {
 
 func sshKeys() error {
 	if !ssh.KeyExists() {
-		spinner := utils.NewSpinner("Generating your client certificates...")
-		spinner.Start()
+		oktetoLog.Spinner("Generating your client certificates...")
+		oktetoLog.StartSpinner()
+		defer oktetoLog.StopSpinner()
 
 		if err := ssh.GenerateKeys(); err != nil {
-			spinner.Stop()
 			return err
 		}
 
-		spinner.Stop()
 		oktetoLog.Success("Client certificates generated")
```

Comment:
With the change, this message could have an iteration of the spinner afterwars. Is that a problem?

Reply:
When having a spinner running and displying TTY logs (in this case a Success Log) we needed to Stop and Start the spinner after. this logic has been moved to the log writter, where we have the hold and unhold functions to perform this actions within the log. This have been removed on every Stop/Start that was made to show the logs.